components/list/list-item-generic-layout.js

```diff
@@ -200,8 +196,8 @@ class ListItemGenericLayout extends RtlMixin(LitElement) {
 			<slot name="outside-control-container"></slot>
 
 			<slot name="content-action" class="d2l-cell" data-cell-num="6"></slot>
-			<slot name="outside-control-action" class="d2l-cell" data-cell-num="1"></slot>
 			<slot name="outside-control" class="d2l-cell" data-cell-num="2"></slot>
+			<slot name="outside-control-action" class="d2l-cell" data-cell-num="1"></slot>
```

Comment:
Yeah, from what I can tell, the `outside-control-action` isn't rendered when the user enters keyboard "move" mode, and for anyone else who might not be familiar, that slot would contain simple `draggable` `div` that overlays the item with some drag & drop event handlers.